src/dug/config.py

```diff
@@ -54,9 +47,50 @@
 }
 
 concept_expander = {
-    'url': "https://tranql.renci.org/tranql/query?dynamic_id_resolution=true&asynchronous=false",
-    'min_tranql_score': 0.0
+    "url": "https://tranql.renci.org/tranql/query?dynamic_id_resolution=true&asynchronous=false",
+    "min_tranql_score": 0.0
 }
 
 # List of ontology types that can be used even if they fail normalization
 ontology_greenlist = ["PATO", "CHEBI", "MONDO", "UBERON", "HP", "MESH", "UMLS"]
+
+
+@dataclass
+class Config:
+    """
+        TODO: Populate description
+    """
+    elastic_password: str = "changeme"
+    redis_password: str = "changeme"
+
+    elastic_host: str = "elasticsearch"
+    elastic_port: int = 9200
+    elastic_username: str = "elastic"
+
+    redis_host: str = "redis"
+    redis_port: int = 6379
+
+    nboost_host: str = "nboost"
+    nboost_port: int = 8000
+
+
+def from_env() -> Config:
+    env_vars = {
+        "elastic_host": "ELASTIC_API_HOST",
+        "elastic_port": "ELASTIC_API_PORT",
+        "elastic_username": "ELASTIC_USERNAME",
+        "elastic_password": "ELASTIC_PASSWORD",
+        "redis_host": "REDIS_HOST",
+        "redis_port": "REDIS_PORT",
+        "redis_password": "REDIS_PASSWORD",
+        "nboost_host": "NBOOST_API_HOST",
+        "nboost_port": "NBOOST_API_PORT"
+    }
+
+    kwargs = {}
+
+    for kwarg, env_var in env_vars.items():
+        if env_value := os.environ.get(env_var):
```

Comment:
Jenkins tests failed since we support Python 3.7, and the `:=` operator isn't available in that version